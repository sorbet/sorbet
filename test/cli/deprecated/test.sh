#!/bin/bash
exec 2>&1
D=test/cli/deprecated

main/sorbet --silence-dev-message $D/t1.rb --enable-experimental-rbs-comments --enable-deprecated
main/sorbet --silence-dev-message $D/t1.rb --enable-experimental-rbs-comments 

main/sorbet --silence-dev-message $D/t2.rb --enable-deprecated
main/sorbet --silence-dev-message $D/t2.rb
