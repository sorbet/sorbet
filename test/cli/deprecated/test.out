test/cli/deprecated/t1.rb:19: Method `BasicDeprecated#old_method` is deprecated https://srb.help/7052
    19 |BasicDeprecated.new.old_method # error: Method `BasicDeprecated#old_method` is deprecated
                            ^^^^^^^^^^
    test/cli/deprecated/t1.rb:9: Defined in `BasicDeprecated` here
     9 |  def old_method
          ^^^^^^^^^^^^^^
Errors: 1
No errors! Great job.
test/cli/deprecated/t2.rb:18: Method `BasicDeprecated#old_method` is deprecated https://srb.help/7052
    18 |BasicDeprecated.new.old_method # error: Method `BasicDeprecated#old_method` is deprecated
                            ^^^^^^^^^^
    test/cli/deprecated/t2.rb:8: Defined in `BasicDeprecated` here
     8 |  def old_method
          ^^^^^^^^^^^^^^
Errors: 1
No errors! Great job.
